apps/acceptance/clock/acceptanceStopwatch.js

```diff
@@ -0,0 +1,71 @@
+'use strict';
+var Clock = require('./regions/appView');
+var expect = require('chai').expect;
+
+marionette('Clock Acceptance Suite', function(){
+    var client;
+    var clock;
+    client = marionette.client({
+        prefs: {
+            // we need to disable the keyboard
+            // to avoid intermittent failures on
+            // Travis (transitions might take
+            // longer to run and block UI)
+            'dom.mozInputMethod.enabled': false,
+            // Do not require the B2G-desktop app
+            // window to have focus (as per the
+            // system window manager) in order for
+            // it to do focus-related things.
+            'focusmanager.testmode': true
+        },
+        settings: {
+            'ftu.manifestURL': null,
+            'lockscreen.enabled': false
+        }
+    });
+    setup(function() {
+        clock = new Clock(client);
+        clock.launch();
+    });
+    suite('Stopwatch Suite', function() {
+        test('Start Stopwatch and Reset', function() {
+            var stopwatchView = clock.switchView('stopwatch');
+
+            stopwatchView.wait(0.2);
+
+            expect(stopwatchView.getCurrentTime()).to.equal('00:00.00');
+
+            stopwatchView.tapStart();
```

Comment:
This is very interesting because this was the route that I was going when I was writing my python tests and I was told that I needed to include actions as they were performed by user rather than intent. I had created higher level functionality but was told that it needed to be a bit more granular. Is this change due to us using JavaScript now? Perhaps conventions of the language? Or are we changing the way we write tests?